src/components/tabs/tabs.module.css

```diff
@@ -24,19 +29,27 @@ ref: https://twitter.com/heydonworks/status/408263890284191745
 ref: https://twitter.com/MarcoInEnglish/status/363284092806119424
 */
 .tabPanel[role='tabpanel'] {
+  /* Note: g-focus-ring-from-box-shadow added in JSX */
+  border-radius: 6px;
```

Comment:
@zchsh I forgot about this until just now, is this supposed to be `6px`? I'm only seeing `5px` in the designs right now